src/Illuminate/Database/Eloquent/Relations/HasOneOrMany.php

```diff
@@ -47,6 +47,19 @@ public function __construct(Builder $query, Model $parent, $foreignKey, $localKe
     }
 
     /**
+     * Create and return an un-saved instance of the related model.
+     *
+     * @param  array  $attributes
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    public function make(array $attributes = [])
+    {
+        return tap($this->related->newInstance($attributes), function ($instance) {
+            $instance->setAttribute($this->getForeignKeyName(), $this->getParentKey());
+        });
```

Comment:
I just copied the `create` method and removed the `save` call. You are right, the `tap` is not needed, as the `$instance` can be chained through the `setAttribute` and then `return`ed.